lib/postgrex/query.ex

```diff
@@ -22,10 +23,13 @@ defmodule Postgrex.Query do
     result_formats: [:binary | :text] | nil,
     decoders:       [Postgrex.Types.oid] | [(binary -> term)] | nil,
     types:          Postgrex.TypeServer.table | nil,
-    null:           atom}
+    null:           atom,
+    max_rows:       integer,
+    state:          :suspended | :done | nil
```

Comment:
not sure what _reusable_ means

is it something like that https://github.com/thenrio/postgrex/commit/810c65e8afd8a7d33d52b5c7a4ee372c5acff8f1#diff-c7c0f9f6218a73a7c8312d9305570b63R709 ?

> `query.state` is actually __portal__ state... name is deceptive

if not then what example for this _reusable_ behavior ?